src/GraphQL.Analyzers.Package/tools/install.ps1

```diff
@@ -0,0 +1,272 @@
+param($installPath, $toolsPath, $package, $project)
```

Comment:
Is this file a typical file copied from somewhere?  The signature block was included with the file, I assume?

Reply:
This file was added by the default template. I didn't really try to understand this script.